docs/product/others/advanced-conversation-filters.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Advanced Conversation Filters"
+---
+
+Chatwoot let's you apply advanced filters for your conversations. You will have the ability to filter conversations with multiple attributes at once and even query them with AND and OR operators.
+
+ offers filtering conversations by the below attributes:
```

Comment:
There is something missing this second paragraph

Reply:
Fixed